### PR TITLE
Filter pull request creator from list of reviewers.

### PR DESCRIPTION
**Pull request by f2prateek**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -69,6 +69,10 @@ app.post('/', function(req, res) {
       data.pull_request.number // 23
     );
 
+    reviewers = reviewers.filter(function(reviewer){
```
Comment (vjeux) [inline server.js]: This is actually not going to work as expected. If the author is part of the top 3 reviewers, then you're only going to have two. Instead you should pass down the author as a third argument of guessOwnersForPullRequest and do the filtering there.

**Comment by vjeux** [comment]
Sweet, thanks for tackling this! I suggested you a better way

File: server.js (hunk 1 above)
Comment (vjeux) [inline server.js]: nit: space before `{`

**Comment by f2prateek** [comment]
(Signed CLA)

File: server.js (hunk 1 above)
Comment (f2prateek) [inline server.js]: Makes sense, I'll update the PR.

**Comment by vjeux** [comment]
Awesome, thanks!

**Comment by vjeux** [comment]
While you are at it, do you want to fix the problem where we sometimes return 'none'. Should be as simple as filtering out the user 'none'

**Pull request merged**